bin/prettier.js

```diff
@@ -210,6 +211,22 @@ function getTrailingComma(argv) {
   }
 }
 
+function getArrowFnParens(argv) {
+  switch (argv["arrow-fn-parens"]) {
+    case "avoid":
```

Comment:
Maybe use fallthrough?

```js
case "avoid":
case "always":
case "functional":
case undefined:
  return argv["arrow-fn-parens"];
```

Reply:
This conforms with surrounding code, though I don't really have an opinion

Reply:
Yeah, I prefer your style but will leave as-is. James wrote the `switch` above so I will conform to that 😄